tests/test_fields.py

```diff
@@ -21,9 +21,9 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import decimal
 import logging
 import unittest
+import decimal
```

Comment:
Style issue: sort imports in this group ("standard imports") alphabetically: decimal, logging, unittest.